sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -58,7 +58,8 @@ trait DataSourceScanExec extends LeafExecNode {
   override def simpleString(maxFields: Int): String = {
     val metadataEntries = metadata.toSeq.sorted.map {
       case (key, value) =>
-        key + ": " + StringUtils.abbreviate(redact(value), 100)
+        key + ": " + StringUtils.abbreviate(redact(value),
```

Comment:
Can you check if there are other places that needs to call `StringUtils.abbreviate` against `simpleString`?
`simpleString` itself, IIRC, abbreviates when it's long unlike `catalogString`.

Reply:
@HyukjinKwon 
StringUtils.abbreviate is used in the two classes below as well : 
    FileScan
    InMemoryRelation
For FileScan, it's also used to abbreviate metadata, so i think we can use maxMetadataValueStringLength there as well.
I just update the code, please review again. @HyukjinKwon

Reply:
Can we replace this place with, for instance, `truncatedString` (see StructType.scala) and reuse `spark.sql.debug.maxToStringFields` configuration?